crates/walrus-simtest/tests/simtest_event_blob.rs

```diff
@@ -65,6 +65,96 @@ mod tests {
         last_blob
     }
 
+    async fn wait_for_event_blob_writer_to_fork(
+        walrus_cluster: &mut TestCluster<SimStorageNodeHandle>,
+        client: &Arc<WithTempDir<Client<SuiContractClient>>>,
+        node_index: usize,
+    ) -> Result<(), anyhow::Error> {
+        let node = &walrus_cluster.nodes[node_index];
+
+        let mut last_certified_blob = get_last_certified_event_blob_must_succeed(client).await;
+        loop {
+            let current_blob = get_last_certified_event_blob_must_succeed(client).await;
+
+            if current_blob.blob_id != last_certified_blob.blob_id {
+                tracing::info!("New event blob seen during fork wait: {:?}", current_blob);
+                last_certified_blob = current_blob;
+                tokio::time::sleep(Duration::from_secs(30)).await;
+
+                let node_blob_id = get_last_certified_event_blob_from_node(node).await?;
+                if node_blob_id != last_certified_blob.blob_id {
```

Comment:
Question: this can also just be that the node is slow and hasn't processed certified event blob?

Reply:
Sorry for not being clear. I understand that it's hard to differentiate slow vs. stuck. I just wonder if you verified in a test case where the node's event blob is indeed stuck.